### PR TITLE
Feature/prot 656

### PR DESCRIPTION
**Pull request by OlegDokuka**
This PR provides updates to the latest changes in proteus-client and provides usage of CustomizableBuilder in order to build connection setup logic

### CODE REVIEW

File: demos/springboot-demo/vowelcount-service/src/main/java/io/netifi/proteus/demo/vowelcount/Application.java [hunk 1]
```diff
@@ -31,14 +31,14 @@ public static void main(String... args) {
...
+//    @Bean
+//    public ProteusConfigurer proteusConfigurer() {
+//        return builder ->
+//            builder.clientTransportFactory(address -> {
+//                TcpClient client = TcpClient.create()
+//                                            .addressSupplier(() -> address);
+//                return TcpClientTransport.create(client);
+//            });
+//
+//    }
```
Comment (robertroeser) [inline demos/springboot-demo/vowelcount-service/src/main/java/io/netifi/proteus/demo/vowelcount/Application.java]: remove the commented code?

Comment (kbahr) [reply demos/springboot-demo/vowelcount-service/src/main/java/io/netifi/proteus/demo/vowelcount/Application.java]: DO we actually not need this? There have been occasions where @OlegDokuka you've commented stuff for testing but actually was necessary ;)

Comment (OlegDokuka) [reply demos/springboot-demo/vowelcount-service/src/main/java/io/netifi/proteus/demo/vowelcount/Application.java]: It is a sample of how to customize proteus, maybe it should not be there at all like it breaks autoconfigured things. Just like a showcase. I can remove it at all or just make kind of identity showcase that log message "called"

Comment (OlegDokuka) [reply demos/springboot-demo/vowelcount-service/src/main/java/io/netifi/proteus/demo/vowelcount/Application.java]: fixed

**Pull request merged**